client/ayon_core/hosts/blender/plugins/publish/validate_deadline_publish.py

```diff
@@ -28,15 +28,27 @@ class ValidateDeadlinePublish(pyblish.api.InstancePlugin,
     def process(self, instance):
         if not self.is_active(instance.data):
             return
+
+        tree = bpy.context.scene.node_tree
+        output_type = "CompositorNodeOutputFile"
+        output_node = None
+        # Remove all output nodes that inlcude "AYON" in the name.
+        # There should be only one.
+        for node in tree.nodes:
+            if node.bl_idname == output_type and "AYON" in node.name:
+                output_node = node
+                break
+        if not output_node:
+            raise PublishValidationError(
+                "No output node found in the compositor tree."
+            )
```

Comment:
You don't need a composite node per se, but you need a composite tree! If you don't require any compositing, it will be a very simple tree and just take the image and output directly:
![image](https://github.com/ynput/ayon-core/assets/1087869/d12f5707-4d54-4d10-a7b0-a392d9420573)
but the compositing tree is still necessary! It is created automatically when you create the instance, and adds the outputs and links for all the render passes selected in the settings.

Reply:
Interesting - thanks for the details. Didn't know that was a Blender requirement. I wonder if this is a bit off an odd design pattern or would be considered nice. 

By the way, this sounds like something that we need documentation for! :)